src/outputs/vtk.cpp

```diff
@@ -242,191 +242,8 @@ void VTKOutput::WriteContainer(Mesh *pm, ParameterInput *pin, bool flag) {
-    std::fprintf(pfile, "\nY_COORDINATES %d float\n", ncoord2);
-    if (ncells2 == 1) {
-      data[0] = static_cast<float>(pmb->pcoord->x2v(out_js));
-    } else {
-      for (int j = out_js; j <= out_je + 1; ++j) {
-        data[j - out_js] = static_cast<float>(pmb->pcoord->x2f(j));
-      }
-    }
-    if (!big_end) {
-      for (int i = 0; i < ncoord2; ++i)
-        Swap4Bytes(&data[i]);
-    }
-    std::fwrite(data, sizeof(float), static_cast<std::size_t>(ncoord2), pfile);
-
-    // write x3-coordinates as binary float in big endian order
-    std::fprintf(pfile, "\nZ_COORDINATES %d float\n", ncoord3);
-    if (ncells3 == 1) {
-      data[0] = static_cast<float>(pmb->pcoord->x3v(out_ks));
-    } else {
-      for (int k = out_ks; k <= out_ke + 1; ++k) {
-        data[k - out_ks] = static_cast<float>(pmb->pcoord->x3f(k));
-      }
-    }
-    if (!big_end) {
-      for (int i = 0; i < ncoord3; ++i)
-        Swap4Bytes(&data[i]);
-    }
-    std::fwrite(data, sizeof(float), static_cast<std::size_t>(ncoord3), pfile);
-
-    //  5. Data.  An arbitrary number of scalars and vectors can be written (every node
-    //  in the OutputData doubly linked lists), all in binary floats format
-    std::fprintf(pfile, "\nCELL_DATA %d", ncells1 * ncells2 * ncells3);
-
-    OutputData *pdata = pfirst_data_;
-    while (pdata != nullptr) {
-      // write data type (SCALARS or VECTORS) and name
-      std::fprintf(pfile, "\n%s %s float\n", pdata->type.c_str(), pdata->name.c_str());
-
-      int nvar = pdata->data.GetDim(4);
-      if (nvar == 1) std::fprintf(pfile, "LOOKUP_TABLE default\n");
-      for (int k = out_ks; k <= out_ke; ++k) {
-        for (int j = out_js; j <= out_je; ++j) {
-          for (int i = out_is; i <= out_ie; ++i) {
-            for (int n = 0; n < nvar; ++n) {
-              data[nvar * (i - out_is) + n] = static_cast<float>(pdata->data(n, k, j, i));
-            }
-          }
-
-          // write data in big endian order
-          if (!big_end) {
-            for (int i = 0; i < (nvar * ncells1); ++i)
-              Swap4Bytes(&data[i]);
-          }
-          std::fwrite(data, sizeof(float), static_cast<std::size_t>(nvar * ncells1),
-                      pfile);
-        }
-      }
-      pdata = pdata->pnext;
-    }
-    // don't forget to close the output file and clean up ptrs to data in OutputData
-    std::fclose(pfile);
-    ClearOutputData(); // required when LoadOutputData() is used.
-    delete[] data;
-    pmb = pmb->next;
-  } // end loop over MeshBlocks
-
-  // increment counters
-  output_params.file_number++;
-  output_params.next_time += output_params.dt;
-  pin->SetInteger(output_params.block_name, "file_number", output_params.file_number);
-  pin->SetReal(output_params.block_name, "next_time", output_params.next_time);
-
-  return;
+void VTKOutput::WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm) {
+  throw std::runtime_error(std::string(__func__) + " is not implemented");
 }
```

Comment:
Are we removing support for these file formats or am I missing something?

Reply:
There was some dead code here.  We can easily add this back in later if it's something we want to support.